### PR TITLE
New release v2.3.3

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

**Comment by axic** [comment]
Would like to include #254, #256 and potentially #172 in this.

**Comment by holgerd77** [comment]
Ah ok, we can postpone then.

**Comment by axic** [comment]
At least #254 as that is ready.

**Comment by axic** [comment]
@holgerd77 do you want to update the changelog with #254 and #256 and do the release?

**Comment by holgerd77** [comment]
@axic Actually didn't want to add these to the changelog, seem to me not relevant enough for library users, but if you want to have this mentioned I'm ok with this too.

Yes, will do the release (or at least re-trigger the tests and latest do the release tomorrow) once #256 tests are through and I can rebase this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to 
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.3.3] - 2018-02-02
+
+- Reworked memory expansion/access for opcodes, PR [#174](https://github.com/ethereumjs/ethereumjs-vm/pull/174) (fixes consensus bugs on
+  large numbers >= 53 bit for opcodes using memory location)
+- Keep stack items as bn.js instances (arithmetic performance increases), PR [#159](https://github.com/ethereumjs/ethereumjs-vm/pull/159)
```
Comment (axic) [inline CHANGELOG.md]: You can include #254 and #256 here (just link to the PRs)

File: README.md [hunk 2]
```diff
@@ -8,15 +8,6 @@
 
 Implements Ethereum's VM in JS
 
-#### Note on Byzantium Support:
```
Comment (axic) [inline README.md]: Why should this be removed?

Comment (axic) [reply README.md]: I think it should be reworded stating it always supports the latest, for compatibility with X chain use version Y.

Comment (holgerd77) [reply README.md]: I added this some time ago with having in mind just to keep this for the transition period, but I also can readd and reword a bit.

Comment (holgerd77) [reply README.md]: @axic I've now updated the text to something more generic now fitting better. If you have some text optimizations/criticism articulate it here, but please approve the PR anyhow, I wouldn't want to run all the tests once again for some text change.

I would the update this on some future occasion on another PR.

**Pull request merged**